espnet/utils/training/batchfy.py

```diff
@@ -20,7 +20,8 @@ def batchfy_by_seq(
     :param str ikey: key to access input (for ASR ikey="input", for TTS, MT ikey="output".)
     :param int iaxis: dimension to access input (for ASR, TTS iaxis=0, for MT iaxis="1".)
     :param str okey: key to access output (for ASR, MT okey="output". for TTS okey="input".)
-    :param int oaxis: dimension to access input (for ASR, TTS, MT iaxis=0, reserved for future research.)
+    :param int oaxis: dimension to access output (for ASR, TTS, MT oaxis=0, reserved for future research,
```

Comment:
We are testing `make_batchset` at https://github.com/espnet/espnet/blob/master/test/test_utils.py#L14
It is nice to add new dummy json for asr_mix.py and expected result there.

This `oaxis` is always -1 if we call this function via `make_batchset`. In fact, I am not the author of this `oaxis`. (@hirofumi0810 is). This `oaxis` feature is too much because nobody can specify this.  How about remove this option?

Reply:
I assigned the "-1" to oaxis in `make_batchset()`. Maybe we should remove the hard assignment and replace it as a parameter in `make_batchset()` as well.

Reply:
That will be great.

Reply:
oaxis means the offset in data[okey]. Note that data is a dict and data[okey] is a list.